src/client/pps.go

```diff
@@ -202,21 +202,6 @@ func NewPipeline(pipelineName string) *pps.Pipeline {
 	return &pps.Pipeline{Name: pipelineName}
 }
 
-// CreatePipelineJob creates and runs a job in PPS.
-// This function is mostly useful internally, users should generally run work
-// by creating pipelines as well.
-func (c APIClient) CreatePipelineJob(pipeline string, outputCommit, statsCommit *pfs.Commit) (*pps.PipelineJob, error) {
```

Comment:
CreatePipelineJob has completely gone away since jobs are now solely created in the transaction-deferred `pps.PropagateJobs`.